scripts/src/main/resources/scripts/command/az

```diff
@@ -40,8 +51,7 @@ function setConfig() {
 
 function doRun() {
```

Comment:
Should we add `setConfig` here in case the `devon.properties` with `export AZURE_CONFIG_DIR` is removed or overwritten? Or is that not necessary?

Reply:
Thanks for your valid question and suggestion.
IMHO we should give users some flexibility. IF they remove this on purpose for a reason we might not think of now, it is not good to always re-add it and getting into the way.